source/mapping-rules/mapping-rule-options/time-series.txt

```diff
@@ -0,0 +1,122 @@
+.. _rm-timeseries:
+
+===========
+Time Series
+===========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure mapping rules to migrate a table that contains a date 
+time column as a :ref:`time series collections <manual-timeseries-collection>`. 
+Time series collections efficiently store time series data. In time 
+series collections, writes are organized so that data from the same 
+source is stored alongside other data points from a similar point in time.
+
+About this Task
+---------------
+
+Compared to normal collections, storing time series data in time series
+collections improves query efficiency and reduces the disk usage for
+time series data and secondary indexes.
+
+Before you Begin
+----------------
+
+For full details on each time series field component, see 
+:ref:`time-series-fields`.
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Select a collection with a date time field 
+
+      a. Navigate to the :guilabel:`Mapping` screen.
+      #. On the :guilabel:`Schema model` pane under the 
+         :guilabel:`MongoDB` header, click on a collection that has a 
+         date time field.
+
+   .. step:: Configure the time-series collection
+     
+      a. On the :guilabel:`Mappings` pane, click the :icon-lg:`Edit` 
+         :guilabel:`Configure` button.
+      #. Select the :guilabel:`Time Series Collection` radio button.
+      #. Select a :guilabel:`Time Field`.
+      #. (Optional) Select a :guilabel:`Meta Field`.
+      #. Select a :guilabel:`Granularity`.
+      #. (Optional) Select a :guilabel:`Expire After Seconds`.
+      #. Click :guilabel:`Save And Close`.
+
+         .. note::
+
+            Once a collection is configured with a time series field, 
+            the :icon-lg:`TimeSeries` icon displays next to the collection
+            name on the :guilabel:`Schema model` pane.
+
+Example
+-------
+
+The following example converts the ``rental_date`` column into a time
+series field in MongoDB:
+
+.. tabs::
```

Comment:
Do you feel the tabs add a lot of value here? As a reader, I feel that it would be helpful to be able to see both the input and output simultaneously to help see the mappings.